protocol/avdl/keybase1/notify_ctl.avdl

```diff
@@ -36,6 +36,10 @@ protocol notifyCtl {
     boolean runtimestats;
     boolean featuredBots;
     boolean saltpack;
+    // Let the service determine if certain notifications can be skipped. Most
+    // useful for the GUI since the service knows what conversation is selected
+    // and can skip updates for things not currently on the screen.
+    boolean allowNotifySkips;
```

Comment:
Sorry to keep sending this one back, but can we change this to `allowChatNotifySkips`?

Reply:
sure but i did this as a general thing intentionally

Reply:
Right but I think the name change ends up making this more general, since now as a subscriber, in theory, I can opt into skipping different categories of stuff.